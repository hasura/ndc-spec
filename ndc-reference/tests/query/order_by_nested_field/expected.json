[
  {
    "rows": [
      {
        "id": 2,
        "location": {
          "country": "Sweden"
        }
      },
      {
        "id": 1,
        "location": {
          "country": "UK"
        }
      }
    ]
  }
]