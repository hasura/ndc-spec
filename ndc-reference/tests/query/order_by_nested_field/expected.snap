---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/order_by_nested_field/request.json
---
[
  {
    "rows": [
      {
        "id": 2,
        "location": {
          "country": "Sweden"
        }
      },
      {
        "id": 3,
        "location": {
          "country": "Sweden"
        }
      },
      {
        "id": 1,
        "location": {
          "country": "UK"
        }
      }
    ]
  }
]
